plugins/rapid7_intsights/icon_rapid7_intsights/util/api.py

```diff
@@ -133,7 +133,11 @@ def takedown_request(self, alert_id: str, target: str) -> dict:
         )
 
     def get_alerts(self, alert_params: AlertParams) -> list:
-        return self.make_request("GET", "public/v1/data/alerts/alerts-list", params=alert_params.to_dict()).json()
+        response = self.make_request("GET", "public/v1/data/alerts/alerts-list", params=alert_params.to_dict())
+        if response.text:
+            return response.json()
```

Comment:
Just in case, wrap this in a try/catch and throw a pluginexception if we can't decode the JSON. Other APIs sometimes return non-json text, even if not originally thought to.